README.md

```diff
@@ -425,6 +429,36 @@ def xor(*args: typing.Union[bytes, typing.Iterable[bytes]]) -> bytes:
     )
 ```
 
+## Basic Math
+
+### Modular Arithmetic
+
+In the example below `9 mod 5` is congruent to `4`, because `4` is the
+remainder when `9` is divided by `5`.
+
+```python
+>>> 9 % 5
+4
+>>> [i % 5 for i in range(10)]
+[0, 1, 2, 3, 4, 0, 1, 2, 3, 4]
+```
+
+Wikipedia [has more information](https://en.wikipedia.org/wiki/Modular_arithmetic).
+
+### Exponentiation
```

Comment:
lets add link to official operator doc

https://docs.python.org/3.10/reference/datamodel.html#emulating-numeric-types
https://docs.python.org/3.10/library/operator.html#operator.pow

Reply:
good call, will do.